educator/src/Dscp/Educator/Web/Educator/Handlers.hs

```diff
@@ -94,17 +94,17 @@ educatorApiHandlers =
       -- Grades
 
     , eGetGrades = \course student assignment isFinalF _since onlyCount ->
-            fmap (mkCountedList onlyCount) $ invoke $
-            educatorGetGrades course student assignment isFinalF
+            invoke $ educatorGetGrades course student assignment isFinalF onlyCount
 
     , eAddGrade = \(NewGrade subH grade) ->
             transact $ educatorPostGrade subH grade
 
       -- Proofs
 
-    , eGetProofs = \pfCourse pfStudent pfAssignment pfOnlyCount ->
-            fmap (mkCountedList pfOnlyCount) $ transact $
-            commonGetProofs def{ pfCourse, pfStudent, pfAssignment }
+    , eGetProofs = \pfCourse pfStudent pfAssignment onlyCount ->
+            fmap (mkCountedList onlyCount) . transact $
+            commonGetProofs
```

Comment:
This is the only function in the group that didn't get rid of `fmap (mkCountedList onlyCount)`
AFAIU if `sGetProofs` implementation also gets modified it should be possible to add the `onlyCount` param to `commonGetProofs`, right?

I noticed `StudentApiEndpoints` have not had this their `onlyCount` uncommented, but `sGetProofs` could just pass a fixed bool for now, I suppose

Reply:
Right, I didn't want to modify `commonGetProofs` because it would affect Student API as well, but your suggestion about using a fixed bool is pretty reasonable, will do that now.

Reply:
Don't you mind if I remain this for another [ticket](https://issues.serokell.io/issue/DSCP-490)?
There seems to be pretty much to think and to do there actually.

Reply:
no problem